src/ripple/protocol/jss.h

```diff
@@ -530,7 +548,9 @@ JSS(server_version);            // out: NetworkOPs
 JSS(settle_delay);              // out: AccountChannels
 JSS(severity);                  // in: LogLevel
 JSS(shards);                    // in/out: GetCounts, DownloadShard
+JSS(sidechain);                 // in: sidechain
```

Comment:
`sidechain`, `signatures`, `was_src_chain_send`, and `xchain_seq` don't appear to be used anywhere in the code.

Reply:
Fixed in c1d0e9e333 [fold] Rm unused jss vars https://github.com/XRPLF/rippled/pull/4292#pullrequestreview-1298384968